src/anf-preview/azext_anf_preview/_params.py

```diff
@@ -0,0 +1,26 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+
+def load_arguments(self, _):
+    with self.argument_context('anf') as c:
+        c.argument('resource_group', options_list=['--resource-group', '-g'], required=True, help='The name of the resource group')
+
+    with self.argument_context('anf') as c:
+        c.argument('account_name', options_list=['--account-name', '-a'], required=True, help='The name of the ANF account')
+
+    with self.argument_context('anf') as c:
+        c.argument('pool_name', options_list=['--pool-name', '-p'], required=True, help='The name of the ANF pool')
+
+    with self.argument_context('anf') as c:
+        c.argument('volume_name', options_list=['--volume-name', '-v'], required=True, help='The name of the ANF volume')
+
+    with self.argument_context('anf') as c:
+        c.argument('snapshot_name', options_list=['--snapshot-name', '-s'], required=True, help='The name of the ANF snapshot')
```

Comment:
By convention, the CLI also has the shorthand option `-n` for the target resource of the command. It would be nice if these parameters did the same. For example, for `az anf account` commands, `--account-name` should also be referred to by `-n`.

Reply:
ok. I will add the -n with a resource specific argument_context.